Slider/tests/Slider-specs.js

```diff
@@ -1,512 +1,441 @@
-		}
-	);
-
-	test(
-		'should decrement the value of vertical slider on key down when active',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} orientation="vertical" />
-			);
-
-			activate(subject);
-			downKeyDown(subject);
-
-			const expected = 49;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should decrement the value of vertical slider on key down',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} orientation="vertical" />
-			);
-
-			focus(subject);
-			downKeyDown(subject);
-
-			const expected = 49;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should increment the value of horizontal slider on key right when active',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} />
-			);
-
-			activate(subject);
-			rightKeyDown(subject);
-
-			const expected = 51;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should increment the value of horizontal slider on key right',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} />
-			);
-
-			focus(subject);
-			rightKeyDown(subject);
-
-			const expected = 51;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should increment the value of vertical slider on key up when active',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} orientation="vertical" />
-			);
-
-			activate(subject);
-			upKeyDown(subject);
-
-			const expected = 51;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
-
-	test(
-		'should increment the value of vertical slider on key up when',
-		() => {
-			const subject = mount(
-				<Slider defaultValue={50} orientation="vertical" />
-			);
+	test('should set "aria-valuetext" to hint string for the first render when vertical is false', () => {
+		render(
+			<Slider />
+		);
+		const slider = screen.getByRole('slider');
 
-			focus(subject);
-			upKeyDown(subject);
-
-			const expected = 51;
-			const actual = subject.find('Slider').prop('value');
-
-			expect(actual).toBe(expected);
-		}
-	);
+		const expectedAttribute = 'aria-valuetext';
+		const expectedValue = '0 change a value with left right button';
 
-	// these tests validate behavior relating to `value` defaulting to `min`
-	test(
-		'should not emit onChange when decrementing at the lower bound when value is unset',
-		() => {
-			const handleChange = jest.fn();
-			const subject = mount(
-				<Slider min={0} max={10} onChange={handleChange} />
-			);
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
+	});
+
+	test('should set "aria-valuetext" to hint string for the first render when vertical is true', () => {
+		render(
+			<Slider orientation="vertical" />
+		);
```

Comment:
let's have the render statement on 1 line on every test

Reply:
Done